src/Messaging/Messaging.ts

```diff
@@ -105,4 +105,31 @@ export class Messaging {
   publishClaimsNotDownloaded = async (ipfsHashFailure: ReadonlyArray<IPFSHashFailure>) => {
     return this.publish(this.exchanges.claimsNotDownloaded, ipfsHashFailure)
   }
+
+  consumeClaimsNotDownloaded = async (consume: (ipfsHashFailures: ReadonlyArray<IPFSHashFailure>) => void) => {
```

Comment:
@lautarodragan not sure if this function is necessary, flip side of `consumeClaimsDownloaded` I know you had some plans to adjust code in this file or something so just alerting you.

Reply:
Thanks @kennylavender.

Placing this function here is fine. The goal is to avoid code duplication given the possibility of having more than one consumer of the same message.

The plans for this file are to [Separate RabbitMQClient and Messaging](https://github.com/poetapp/node/issues/66).